recipes/pulp/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.6.8" %}
+
+package:
+  name: pulp
+  version: '{{ version }}'
+
+source:
+  sha256: cfecd6a78c2b08539dc29b7e8281f43d729b1089bdb5e491c3c0956aadbb3fc7
+  fn: 'PuLP-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/P/PuLP/PuLP-{{ version }}.tar.gz
+
+build:
+  entry_points:
+    - pulptest=pulp:pulpTestAll
+    - pulpdoctest=pulp:pulpDoctest
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyparsing >=2.0.1
+  run:
+    - python
+    - pyparsing >=2.0.1
+
+test:
+  commands:
+    - pulptest --help
+  imports:
+    - pulp
+    - pulp.solverdir
+
+about:
+  summary: |
+    PuLP is an LP modeler written in python. PuLP can generate MPS or LP files
+    and call GLPK, COIN CLP/CBC, CPLEX, and GUROBI to solve linear problems.
+  home: https://github.com/coin-or/pulp
+  license: MIT
+  doc_url: https://pythonhosted.org/PuLP/
+  dev_url: https://github.com/coin-or/pulp
+  license_family: MIT
```

Comment:
Add `license_file: LICENSE`, please.

Reply:
I don't think MIT requires this. But ok.

Reply:
MIT license says "The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software." I think source distributions definitely need it, and [it's ambiguous](https://github.com/github/choosealicense.com/issues/257) whether compiled versions do—but for Python code, you definitely have to include the license file.

Reply:
Indeed, you are right I guess ;-).